plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSessionProperties.java

```diff
@@ -54,6 +56,7 @@ public class HudiSessionProperties
     private static final String SPLIT_GENERATOR_PARALLELISM = "split_generator_parallelism";
     static final String QUERY_PARTITION_FILTER_REQUIRED = "query_partition_filter_required";
     private static final String IGNORE_ABSENT_PARTITIONS = "ignore_absent_partitions";
+    private static final String DYNAMIC_FILTERING_WAIT_TIMEOUT = "dynamic_filtering_wait_timeout";
```

Comment:
Similarly, if this is in the unit of seconds, add `_seconds` to the config name.  If it takes the duration format like `1s` then there is no need to rename the config.

Reply:
Takes a integer unit. This was copied over from Hive's and Delta's implementation, here:

https://github.com/onehouseinc/trino/blob/92ebcc4216269764c4a70d6eb4d4366a9210109d/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSessionProperties.java#L122-L124


Nonetheless, will change it.

Reply:
It's a duration type, standardizing with the rest of the engines.

Reply:
Got it.  Sg.